Homework_2.0.py

```diff
@@ -103,7 +80,8 @@ def display_info():
 
 @app.route("/add_name", methods=["POST"])
 def add_name():
-    item = request.form.get('item')
+    # присваиваем имя инстанса
+    item = request.form.get('name')
```

Comment:
It is a bad practice to use the same variable name in your case `item` in all your functions, it could mislead you in development process